components/common-go/log/log.go

```diff
@@ -36,6 +40,35 @@ func OWI(owner, workspace, instance string) log.Fields {
 	}
 }
 
+// LogContext builds a structure meant for logrus which contains the owner, workspace and instance.
+// Beware that this refers to the terminology outside of wsman which maps like:
+//    owner = owner, workspace = metaID, instance = workspaceID
+func LogContext(owner, workspace, instance, project, team string) log.Fields {
```

Comment:
I think we could extend this function https://github.com/gitpod-io/gitpod/blob/e1b335c029047debce6209485635c76b678a5e36/components/common-go/log/log.go#L31 to have the project and team as well.

Or to create another function
```go
func PT(project, team string) log.Fields {
	return log.Fields{
		Project: project,
		Team: team,
	}
}
```

Reply:
Well, my idea was has a another function like this
```go
func PT(project, team string) log.Fields {
	return log.Fields{
		Project: project,
		Team: team,
	}
}
```

So, the caller would be
```go
owi := log.OWI(req.Metadata.Owner, req.Metadata.MetaId, req.Id)
pt := log.PT(project, team)
clog := log.WithFields(owi).WithFields(pt)
```